src/box/key_def.h

```diff
@@ -310,24 +310,26 @@ enum {
 };
 
 /**
```

Comment:
If we'll not add a documentation comment to the constant itself, it'll look so:

![image](https://user-images.githubusercontent.com/92749/227655892-04ece622-ed92-408f-b47e-587ec7e8086b.png)

I propose the following:

```c
enum {
        /** The constant. */
        BOX_KEY_PART_DEF_T_SIZE = 64,
};
```

It looks so:

![image](https://user-images.githubusercontent.com/92749/227655976-e571569b-4bef-4fab-92da-6a927307a977.png)

Reply:
Looks much better! Updated, thanks.